debian/control

```diff
@@ -25,7 +25,6 @@ Description: Endless boot helper
 Depends: ${misc:Depends},
          dracut,
          xinit,
```

Comment:
Isn't the `xinit` dependency actually only for `eos-factory-tools` too?

Reply:
Indeed, updated - checked and `eos-factory-tools` already includes it.